README.md

```diff
@@ -1,6 +1,7 @@
 # Reaction
 
-[![bitHound Overall Score](https://www.bithound.io/github/reactioncommerce/reaction/badges/score.svg)](https://www.bithound.io/github/reactioncommerce/reaction) [![bitHound Dev Dependencies](https://www.bithound.io/github/reactioncommerce/reaction/badges/devDependencies.svg)](https://www.bithound.io/github/reactioncommerce/reaction/9a858eb459d7260d5ae59124c2b364bc791a3e70/dependencies/npm) [![bitHound Code](https://www.bithound.io/github/reactioncommerce/reaction/badges/code.svg)](https://www.bithound.io/github/reactioncommerce/reaction) [![Code Climate](https://codeclimate.com/github/reactioncommerce/reaction/badges/gpa.svg)](https://codeclimate.com/github/reactioncommerce/reaction) [![Circle CI](https://circleci.com/gh/reactioncommerce/reaction.svg?style=svg)](https://circleci.com/gh/reactioncommerce/reaction) [![Gitter](https://badges.gitter.im/JoinChat.svg)](https://gitter.im/reactioncommerce/reaction?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![bitHound Overall Score](https://www.bithound.io/github/andela/jiayou-rc/badges/score.svg)](https://www.bithound.io/github/andela/jiayou-rc) [![bitHound Dev Dependencies](https://www.bithound.io/github/andela/jiayou-rc/badges/devDependencies.svg)](https://www.bithound.io/github/andela/jiayou-rc/develop/dependencies/npm) [![bitHound Code](https://www.bithound.io/github/andela/jiayou-rc/badges/code.svg)](https://www.bithound.io/github/andela/jiayou-rc) [![Code Climate](https://codeclimate.com/github/andela/jiayou-rc/badges/gpa.svg)](https://codeclimate.com/github/andela/jiayou-rc) [![Circle CI](https://circleci.com/gh/andela/jiayou-rc.svg?style=svg)](https://circleci.com/gh/andela/jiayou-rc) 
```

Comment:
Circle CI report status doesn't show on the branch. please, it's important for us to have all the branches showing their status based on the last build. Why are you pointing `bithound.io` to develop branch